project_report/tex/requirements.tex

```diff
@@ -0,0 +1,76 @@
+\section{Requirements}
+
+\subsection{Overall Requirements Specification}
+
+\subsection{Selected Detailed Requirements}
+
+\subsubsection{Functional \& Non-Functional Requirements}
+In table \ref{table:Sup_requirements} you can see a description of Functional \&
+Non-Functional requirements showed in the model of FURPS+. The Model FURPS+ 
+gives a good overview of which Functional \& Non-Functional requirements their is.
```

Comment:
Delete "The Model FURPS+ gives a good overview of which Functional \& Non-Functional requirements their is."